.depcheckrc.json

```diff
@@ -1,11 +1,12 @@
 {
+  "ignore-patterns": [".yarn"],
   "ignores": [
     "@lavamoat/allow-scripts",
     "@lavamoat/preinstall-always-fail",
-    "@metamask/auto-changelog",
```

Comment:
We don't need to ignore `@metamask/auto-changelog` now?

Reply:
No, `depcheck` seems to detect that it's used now.